.github/workflows/bundle_size.yml

```diff
@@ -0,0 +1,33 @@
+name: Bundle Size
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: ./code
+
+jobs:
+  bundle-size:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm ci
+      - name: Analyze bundle size
+        run: npx shadow-cljs run shadow.cljs.build-report nuvla-ui bundle-size-report-${{ github.sha }}.html
+      - uses: actions/upload-artifact@v3
```

Comment:
Seems that there is a problem with the upload: https://github.com/nuvla/ui/actions/runs/3412853723

Reply:
Worked now!